README.md

```diff
@@ -83,13 +103,29 @@ env:
   HYDROLIX_PASSWORD: <hydrolix-password>
 ```
 
+MCP Server definition using service account token (YAML):
+
+```yaml
+command: uv
+args:
+- run
+- --with
+- mcp-hydrolix
+- --python
+- "3.13"
+- mcp-hydrolix
+env:
+  HYDROLIX_HOST: <hydrolix-host>
+  HYDROLIX_TOKEN: <hydrolix-service-account-token>
+```
+
 ### Configuration Example (Claude Desktop)
 
 1. Open the Claude Desktop configuration file located at:
    - On macOS: `~/Library/Application Support/Claude/claude_desktop_config.json`
    - On Windows: `%APPDATA%/Claude/claude_desktop_config.json`
 
-2. Add a `mcp-hydrolix` server entry to the `mcpServers` config block:
+2. Add a `mcp-hydrolix` server entry to the `mcpServers` config block to use username and password:
```

Comment:
I think we should revert this line and simply remove the user/password example from this bullet point -- we already have exhaustive examples above, this is just a single arbitrary example -- and I'd expect service account to be the more common case.